pkg/minikube/cruntime/docker.go

```diff
@@ -692,17 +697,19 @@ func dockerConfigureNetworkPlugin(r Docker, cr CommandRunner, networkPlugin stri
 	}
 
 	opts := struct {
+		PauseImage     string
 		NetworkPlugin  string
 		ExtraArguments string
 	}{
+		PauseImage:     pauseImage,
 		NetworkPlugin:  networkPlugin,
 		ExtraArguments: args,
 	}
 
 	const CRIDockerServiceConfFile = "/etc/systemd/system/cri-docker.service.d/10-cni.conf"
 	var CRIDockerServiceConfTemplate = template.Must(template.New("criDockerServiceConfTemplate").Parse(`[Service]
 ExecStart=
-ExecStart=/usr/bin/cri-dockerd --container-runtime-endpoint fd:// --network-plugin={{.NetworkPlugin}}{{.ExtraArguments}}`))
+ExecStart=/usr/bin/cri-dockerd --container-runtime-endpoint fd:// --pod-infra-container-image={.PauseImage} --network-plugin={{.NetworkPlugin}}{{.ExtraArguments}}`))
```

Comment:
Typo?
```suggestion
ExecStart=/usr/bin/cri-dockerd --container-runtime-endpoint fd:// --pod-infra-container-image={{.PauseImage}} --network-plugin={{.NetworkPlugin}}{{.ExtraArguments}}`))
```

Context: Trying out this PR to see if it fixes #14724 and it's failing for me.

`kubelet` reports this error early on, which I suspect is the root cause of the many other errors it reports:
```
Aug 04 12:13:45 MYHOSTNAME kubelet[4121867]: E0804 12:13:45.272967 4121867 remote_runtime.go:212] "RunPodSandbox from runtime service failed" err="rpc error: code = Unknown desc = failed to inspect sandbox image \"{.PauseImage}\": Error response from daemon: no such image: {.PauseImage}: invalid reference format: repository name must be lowercase"
```

Making the suggested change results in a working minikube for me.